jbpm-bom/pom.xml

```diff
@@ -815,6 +815,12 @@
         <version>${version.org.kie}</version>
         <classifier>sources</classifier>
       </dependency>
+      <dependency>
+        <groupId>org.jbpm</groupId>
+        <artifactId>jbpm-wb-case-mgmt-showcase</artifactId>
+        <version>${version.org.kie}</version>
+        <type>war</type>
+      </dependency>
```

Comment:
This case mgmt interface shouldn't really be treated differently from that point of view, it should be tested and will be delivered as part of the workbench.  Technically however, it is not hardcoded in the workbench but delivered as a separate war that can be provisioned into the workbench.